quick_idea_generation.py

```
@@ -0,0 +1,63 @@
+    
+    result = await orchestrator.run_qadi_cycle(
+        problem_statement="How to live longer",
+        context="""
+        Consider multiple approaches:
+        - Lifestyle and behavioral changes
+        - Medical and technological interventions
+        - Social and environmental factors
+        - Preventive healthcare strategies
+        - Mental and emotional well-being
+        """,
+        cycle_config={"max_ideas_per_method": 3}
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)